native/src/main/java/org/ballerinax/asb/admin/Administrator.java

```diff
@@ -0,0 +1,999 @@
+            LOGGER.debug("Retrieved topic successfully with name: " + topicProp.getName());
+            return constructTopicCreatedRecord(topicProp);
+        } catch (BError e) {
+            return ASBErrorCreator.fromBError(e);
+        } catch (HttpResponseException e) {
+            return ASBErrorCreator.fromASBHttpResponseException(e);
+        } catch (ServiceBusException e) {
+            return ASBErrorCreator.fromASBException(e);
+        } catch (Exception e) {
+            return ASBErrorCreator.fromUnhandledException(e);
+        }
+    }
+
+    /**
+     * Update a Topic in Azure Service Bus.
+     *
+     * @param administratorClient Azure Service Bus Administrator Client
+     * @param topicName           name of the topic
+     * @param topicProperties     properties of the Topic (Requires TopicProperties object)
+     * @return topicProperties
+     */
+    public static Object updateTopic(BObject administratorClient, BString topicName,
+                                     BMap<BString, Object> topicProperties) {
+        ServiceBusAdministrationClient clientEp = getAdminFromBObject(administratorClient);
+        try {
+            TopicProperties topicProp = clientEp.getTopic(topicName.toString());
+            TopicProperties updatedTopicProps = clientEp.updateTopic(
+                    ASBUtils.getUpdatedTopicPropertiesFromBObject(topicProperties, topicProp));
+            LOGGER.debug("Updated topic successfully with name: " + updatedTopicProps.getName());
+            return constructTopicCreatedRecord(updatedTopicProps);
+        } catch (BError e) {
+            return ASBErrorCreator.fromBError(e);
+        } catch (HttpResponseException e) {
+            return ASBErrorCreator.fromASBHttpResponseException(e);
+        } catch (ServiceBusException e) {
+            return ASBErrorCreator.fromASBException(e);
+        } catch (Exception e) {
+            return ASBErrorCreator.fromUnhandledException(e);
+        }
+
+    }
+
+    /**
+     * Get all Topics in Azure Service Bus.
+     *
+     * @param administratorClient Azure Service Bus Administrator Client
+     * @return topicProperties
+     */
+    public static Object listTopics(BObject administratorClient) {
+        ServiceBusAdministrationClient clientEp = getAdminFromBObject(administratorClient);
+        try {
+            PagedIterable<TopicProperties> topicProp = clientEp.listTopics();
+            LOGGER.debug("Retrieved all topics successfully");
+            return constructTopicPropertiesArray(topicProp);
+        } catch (BError e) {
+            return ASBErrorCreator.fromBError(e);
+        } catch (HttpResponseException e) {
+            return ASBErrorCreator.fromASBHttpResponseException(e);
+        } catch (ServiceBusException e) {
+            return ASBErrorCreator.fromASBException(e);
+        } catch (Exception e) {
+            return ASBErrorCreator.fromUnhandledException(e);
+
```

Comment:
additional blank line

Reply:
Addressed in [9882b4a](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/180/commits/9882b4a8780e73f7b894e0df833a9acf7abeca4d).